IBaseEntity/IBaseEntity.cpp

```diff
@@ -25,69 +27,11 @@ IBaseEntity::IBaseEntity() {
     m_tConstructionTime = g_pGlobals->curtime;
 
     m_tLastTimeUsed = -FLT_MAX;
-
-    // in a cooked game PostDuplicate is not called so let's call it here
-    // we'll also call it if all other other BeginPlays have been called
-    if (IsCookedBuild() || g_pGlobals->worldcreated) AddEntityToLists(this);

```

Comment:
Again, no need for this since it shouldn't be that we have two separate code cases for whether the game is cooked or not - why not have it all in one method?